oeps/architectural-decisions/oep-0014-proc-archive-repos.rst

```diff
@@ -27,31 +27,34 @@ OEP-14: Archiving edX GitHub Repositories
 
 .. _ORA PR Discussion: https://github.com/edx/edx-ora/pull/187
 .. _Initial Archiving Discussions: https://openedx.atlassian.net/wiki/display/IT/Proposed+Github+Deprecation+Process
+
+**Code Review: This link is private. Could someone move it to the COMM space or suggest a better alternative?**
+
 .. _Moving More Repos to edx-unsupported: https://openedx.atlassian.net/browse/ARCHBOM-1481
 
 Abstract
 ========
 
-The `edx organization`_ contains a large number of repositories, most of which are active and maintained, but some of which are now obsolete. To clarify the status of repositories, a process for archiving a repository is defined below.
+The `openedx organization`_ contains a large number of repositories, most of which are active and maintained, but some of which are now obsolete. To clarify the status of repositories, a process for archiving a repository is defined below.
```

Comment:
Can we write this OEP so that it can be applied to many GitHub organizations?  I'd hate to see this apply only to openedx, and then have edX either have no process, or have to copy this document to have a process.

Reply:
I think edX/internal deprecations tend to be handled via our Jira boards. The DEPR process for better or worse is primarily a means for announcing deprecations to the wider community.

Reply:
@nedbat I would be comfortable adding a line like, "we encourage others to follow these guidelines", but I'm not sure how I feel about an OEP specifically calling out a community member to say, hey this is your guideline too. Why can't edX just use this guideline? Maybe there's an internal wiki page that says "Hey we follow this oep" and points here.

Reply:
I'm not suggesting to call out a community member.  I want edX to use this guideline.  I'm just suggesting using broader language than "The openedx organization".  How about "The Open edX project, including the openedx GitHub organization, contains a large number ..."

Reply:
I feel that's saying that every Open edX project must use this process...?

Reply:
In my mind "the code of the Open edX project" is equivalent to the code in the `openedx` GitHub organization. There are certainly Open edX plugins, tooling, config overrides, etc other GitHub orgs, but I don't think they need to be governed by Open edX's own project policies. Like @sarina said, if edX wants to opt-in to this OEP for their GitHub org, that's awesome, but I wouldn't want it to seem like we're trying to tell all Open-edX-associated organizations how to manage their repositories.

There are definitely some Open edX project policies that could span multiple GitHub orgs - the Code of Conduct comes to mind. But that's a social policy, whereas this is a very tactical OEP about how exactly to archive a repository.

Reply:
OK, we can adopt the OEP by reference.